packages/xrandr.rb

```diff
@@ -0,0 +1,27 @@
+require 'buildsystems/autotools'
+
+class Xrandr < Autotools
+  description 'Command line interface to X11 Resize, Rotate, and Reflect (RandR) extension'
+  homepage 'https://gitlab.freedesktop.org/xorg/app/xrandr'
+  version '1.5.3'
+  license 'Copyright © 2001 Keith Packard, member of The XFree86 Project, Inc.
```

Comment:
The license is MIT-- these are just the authors listed at the top of the file.

Reply:
I'm not sure how you can deduct that from the COPYING doc but I made the change.

Reply:
For posterity's sake-- thats the MIT license format, this one just has multiple copyright holders. Also, when in doubt, checking repology is usually helpful: https://repology.org/project/xrandr/information